velox/exec/TableWriter.cpp

```diff
@@ -41,7 +41,8 @@ TableWriter::TableWriter(
   // partition
   const auto& connectorId = tableWriteNode->insertTableHandle()->connectorId();
   connector_ = connector::getConnector(connectorId);
-  connectorQueryCtx_ = driverCtx_->createConnectorQueryCtx(connectorId);
+  connectorQueryCtx_ =
+      driverCtx_->createConnectorQueryCtx(connectorId, "TableWriter");
```

Comment:
This is strange. The second parameter is supposed to be the ID of the TableScan node, but it is "TableWriter".